src/colorizer/RecordingControls.ts

```diff
@@ -0,0 +1,149 @@
+import ColorizeCanvas from "./ColorizeCanvas";
+
+export default class RecordingControls {
+  private startBtn: HTMLButtonElement;
+  private abortBtn: HTMLButtonElement;
+  private recording: boolean;
+  private startAtCurrentFrameChkbx: HTMLInputElement;
+
+  private filePrefixInput: HTMLInputElement;
+  private filePrefixResetBtn: HTMLButtonElement;
+  private useDefaultPrefix: boolean;
+  private defaultPrefix: string;
+  private hiddenAnchorEl: HTMLAnchorElement;
+
+  private timerId: number;
+  private startingFrame: number;
+  private redrawfn: () => void;
+  private isDisabled: boolean;
+
+  private canvas: ColorizeCanvas;
+
+  constructor(canvas: ColorizeCanvas, redrawfn: () => void) {
+    this.startBtn = document.querySelector("#sequence_start_btn")!;
+    this.abortBtn = document.querySelector("#sequence_abort_btn")!;
+
+    this.filePrefixResetBtn = document.querySelector("#sequence_prefix_reset_btn")!;
+    this.startAtCurrentFrameChkbx = document.querySelector("#sequence_start_frame_checkbox")!;
+    this.filePrefixInput = document.querySelector("#sequence_prefix")!;
+    this.useDefaultPrefix = true;
+    this.defaultPrefix = "";
+    
+    this.hiddenAnchorEl = document.createElement("a");
+    document.body.appendChild(this.hiddenAnchorEl);
+    
+    this.recording = false;
+    this.canvas = canvas;
+
+    this.timerId = 0;
+    this.startingFrame = 0;
+    this.redrawfn = redrawfn;
+    this.isDisabled = false;
+
+    this.startBtn.addEventListener("click", () => this.handleStartButtonClick());
+    this.abortBtn.addEventListener("click", () => this.handleAbortButtonClick());
+    this.filePrefixInput.addEventListener("changed", () => this.handlePrefixChanged());
+    this.filePrefixResetBtn.addEventListener("click", () => this.handlePrefixResetClicked());
```

Comment:
Minor nitpick: I suggest grouping all the `queryselector` and `addeventlistener` lines together so that the ui elements are clearly logically distinct from the rest of the state.  Also that would match how TimeControls looks.